### PR TITLE
New Theme

### PR DESCRIPTION
**Pull request by Raptorial**

### CODE REVIEW

**Comment by mwiencek** [comment]
There seems to be some problems with this. When I run it locally, this is what I see (after compiling the CSS):

Image showing issues with new theme

Were you seeing something different?

File: website/frontend/templates/changelog.html [hunk 1]
```diff
@@ -27,5 +27,5 @@
         {% endif %}
       {% endfor %}
       </div>
-    </div><!-- /.container -->
```
Comment (mwiencek) [inline website/frontend/templates/changelog.html]: Why is this being moved to the end of the file? You've done this in all the files.

**Comment by mwiencek** [comment]
The change to website/frontend/static/img/picard-icon.png seems wrong, since it went from a large image to a really small one, unless it's being used that way now.

**Comment by mwiencek** [comment]
How come the screenshot was removed in 9bf1deb75da0990f2086e4cffc82ba3e5ca06507?

**Comment by mwiencek** [comment]
Sorry, regarding the screenshot I posted, I figured out it's because I'm running `fab compile_styling` first, which overwrites the CSS you added.

I think us having styles.css checked-in was a mistake, and it shouldn't be modified. Instead, only the .less files under https://github.com/musicbrainz/picard-website/tree/master/website/frontend/static/less should be changed.

**Comment by gentlecat** [comment]
I'm not even sure why CSS is in the repo. Might be better to ignore it.

Anyway, that's a good start, though some work is still needed. Here are some issues I noticed:
* It would be best to preserve original layout and content for now (screenshot, etc.)
* Content in blocks that describe features on homepage should be centered, as it was in the old version.
* It is better to use vector images (see https://github.com/metabrainz/metabrainz-logos/tree/master/final-logos/Picard/VECTOR/SVG) instead of PNGs. They are smaller and render better on high-dpi screens.
* Width of logo block in navbar should be smaller. To determine exact size you can insert it into the page, look at width of the image and specify it like this in *.less* file: https://github.com/metabrainz/metabrainz.org/blob/master/metabrainz/static/css/main.less#L23 (first number is width of the image, which you need to modify).

**Comment by gentlecat** [comment]
For modifying Less code, you can get new theme from, let's say, MetaBrainz website repo at https://github.com/metabrainz/metabrainz.org/tree/master/metabrainz/static/css (*theme* directory). It should be a good starting point which you can combine with existing styling there.

**Comment by mwiencek** [comment]
Superseded by https://github.com/musicbrainz/picard-website/pull/37

**Pull request closed without merging**